students/chaycasso/Lesson03/mailroom1.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+#
+# Assignment: Mailroom, Part 1
+# Chay Casso
+# 2/10/2018
+
+# Initial donor table with the donation values.
+donor_table = [["William Gates, III", 401321.52], ["William Gates, III", 201342.71], ["Mark Zuckerberg", 123.45],
+               ["Mark Zuckerberg", 5123.21], ["Mark Zuckerberg", 8213.11], ["Jeff Bezos", 877.33],
+               ["Paul Allen", 152.42], ["Paul Allen", 30.54], ["Paul Allen", 825.21], ["Steve Ballmer", 5198.96],
+               ["Steve Ballmer", 654.98]]
+answer = ""
+
+
+def thank_you(donor_table):
+    full_name = ""
+    while full_name.lower() != "quit":
+        full_name = input("Please enter a full name. >")
+        if full_name.lower() == "quit": return(donor_table)
+        if full_name.lower() == "list":
+            donor_table.sort()
+            current_name = ""
+            print("Donor list:")
+            for row in donor_table:
+                if current_name != row[0]:
+                    current_name = row[0]
+                    print(current_name)
+        else:
+            donation_value_str = input("Please enter a donation amount. >")
+            if donation_value_str.lower() == "quit": return(donor_table)
+            donation_value = float(donation_value_str)
+            donor_table.append([full_name, donation_value])
```

Comment:
you could do:
```
donor_table.append([full_name, float(donation_value_str)])
```
and not need an extra variable.

Reply:
You also might want a `try except` here in case the user doesn't enter a number.